src/templates.js

```diff
@@ -53,7 +53,7 @@ function renderLayer (featureCollection = {}, options = {}) {
   if (json.drawingInfo) json.drawingInfo.renderer = renderers[json.geometryType]
   if (json.timeInfo) json.timeInfo = metadata.timeInfo
   if (json.maxRecordCount) json.maxRecordCount = metadata.maxRecordCount || 2000
-  if (json.displayField) json.displayField = metadata.displayField || json.fields[0].name
+  if (json.displayField) json.displayField = metadata.displayField || json.fields[0].name || json.displayField
```

Comment:
this will still error out if json.fields is null

Reply:
Now updated with `_.get(json, 'fields[0].name')`

Reply:
lodash ftw, let's just add a test here.

Reply:
Added tests for `fields: null` and `fields: []`.